docs/recipes/code-coverage.md

```diff
@@ -170,6 +170,28 @@ after_success:
 
 Your coverage report will then appear on coveralls shortly after Travis completes.
 
+## Integrating with AVA's Watch mode
+
+While AVA's watch mode [won't](https://github.com/avajs/ava/issues/1100) integrate with `nyc` directly out of the box, you can achieve that using [nodemon](https://github.com/remy/nodemon).
+
+First, install `nodemon` as a dev dependency:
+
+```bash
+$ npm install nodemon --save-dev
+```
+
+Then, create a new npm scripts entry, let's say, `watch`:
+
+```json
+"scripts": {
+	"coverage": "nyc report --reporter=text-lcov | coveralls",
```

Comment:
I considered your point, slept over it and considered again. This line is called by `.travis.yml`. Without it, Travis won't be able to ping Coveralls. We _can_ omit it, but then we should note aside that `scripts` snippet is incomplete. Some people might think it's complete, paste `gung-ho` and their Coveralls won't update. Being a lazy person, I like complete snippets, ones without missing bits.

I'm happy to remove this line, just say no and I'll remove it.